.rubocop.yml

```diff
@@ -78,40 +83,25 @@ Style/ClassAndModuleChildren:
 Metrics/ClassLength:
   Enabled: false
 
-Layout/IndentationWidth:
-  Enabled: false
-
 Layout/EndAlignment:
-  Enabled: false
-
-Layout/ElseAlignment:
-  Enabled: false
-
-Performance/ChainArrayAllocation:
-  Enabled: true
+  EnforcedStyleAlignWith: start_of_line
 
 Style/BlockDelimiters:
   Enabled: false
 
 Style/RedundantSelf:
   Enabled: false
 
-Style/GuardClause:
-  Enabled: false
-
 Style/SymbolArray:
   Enabled: false
 
 Style/WordArray:
   Enabled: false
 
-Style/ParallelAssignment:
-  Enabled: false
-
-Style/PreferredHashMethods:
-  EnforcedStyle: verbose
+Style/RaiseArgs:
+  EnforcedStyle: compact
```

Comment:
Except 4 out of 69, all our `raise`s are in the default form. If  you want to switch the style, we need to replace all the raises so that the code will not be "red-lined" everywhere.